admin_tests/nodes/test_views.py

```diff
@@ -141,13 +141,16 @@ def test_remove_node(self):
         nt.assert_equal(AdminLogEntry.objects.count(), count + 1)
 
     def test_restore_node(self):
-        self.view.delete(self.request)
+        # Removing node outside of admin app
+        self.node.remove_node(auth=Auth(self.node.creator))
```

Comment:
Removing the node in the test so all the typical "remove node" behavior is adhered to, and we can test that restoring converted the root folder's TrashedFolder type to an OsfStorageFolder.  

(Important then, is that removing the node in the admin app does *not* call `Node.remove_node()` so typical node removal behavior isn't followed, the node is just marked as deleted. Is this intentional? Under what use case are we marking nodes as deleted in the admin app?  We might want to refactor so *removing* a node in the admin app follows normal procedures.

Reply:
The only reason I could think to do it like that is for Spam (delete first, ask questions later kind of thing), but if we can restore then it's not a big deal. If we can't be sure of restoring to the proper state (because we don't know which files to undelete in the case of files previously deleted by the user mixing with files deleted by this functionality) then we might want to consider how to make this more robust?